letsencrypt/config.yaml

```diff
@@ -66,7 +67,11 @@ schema:
     ovh_consumer_key: str?
     ovh_endpoint: str?
     propagation_seconds: int(60,3600)?
-    provider: list(dns-cloudflare|dns-cloudxns|dns-digitalocean|dns-directadmin|dns-dnsimple|dns-dnsmadeeasy|dns-gehirn|dns-google|dns-linode|dns-luadns|dns-njalla|dns-nsone|dns-ovh|dns-rfc2136|dns-route53|dns-sakuracloud|dns-netcup|dns-gandi|dns-transip)?
+    provider: "list(dns-cloudflare|dns-cloudxns|dns-digitalocean|\
```

Comment:
Can you use `>-` with multiline here to make it more readable?

Reply:
I don't think this works, `>-` only strips newlines at the end (according to https://yaml-multiline.info/)

Reply:
`|-` should work